HenBot/RepositoryContext.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace HenBot;
+
+public class RepositoryContext : DbContext
+{
+    public DbSet<SavedUser>? SavedUsers { get; set; }
+    public DbSet<TagQuery>? TagQuery { get; set; } 
+
+    public string DbPath { get; }
+
+    public RepositoryContext()
+    {
+        var folder = Environment.SpecialFolder.LocalApplicationData;
+        var path = Environment.GetFolderPath(folder);
+        DbPath = Path.Join(path, "users.db");
+    }
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder) 
+        => optionsBuilder.UseSqlite($"Data Source={DbPath}");
```

Comment:
FYI
You can add query logging with adding package 'Microsoft.Extensions.Logging.Console'

```csharp
protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        optionsBuilder
            .UseLoggerFactory(LoggerFactory.Create(builder => builder.AddConsole()))
            .EnableSensitiveDataLogging()
            .UseSqlite($"Data Source={DbPath}");
        base.OnConfiguring(optionsBuilder);
    }
```

It would also be nice to add console logging only if some specific value is in your system envs or the application is run with some debug key